include/TrustWalletCore/TWHDWallet.h

```diff
@@ -49,6 +49,10 @@ TWData *_Nonnull TWHDWalletSeed(struct TWHDWallet *_Nonnull wallet);
 TW_EXPORT_PROPERTY
 TWString *_Nonnull TWHDWalletMnemonic(struct TWHDWallet *_Nonnull wallet);
 
+/// Returns master key.
+TW_EXPORT_METHOD
+struct TWPrivateKey *_Nonnull TWHDWalletGetMasterKey(struct TWHDWallet *_Nonnull wallet, enum TWCurve curve);
```

Comment:
is master key needed?

Reply:
see: https://github.com/trustwallet/wallet-core/pull/754#discussion_r354062662